RHEL/6/input/checks/avahi_disable_publishing.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
+  <definition class="compliance" id="avahi_disable_publishing" version="1">
+    <metadata>
+      <title>Disable Avahi Publishing</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 6</platform>
+      </affected>
+      <description>Check that Avahi is ignoring packets
+      unless the TTL field is 255.
+      </description>
```

Comment:
This description is the same as for the 'avahi_check_ttl' check. Please update it.